File: 151001/Danko/RESTApplication/src/main/java/com/example/restapplication/dto/MessageRequestTo.java
package com.example.restapplication.dto;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MessageRequestTo {
    private Long id;
    private Long storyId;
    @NotBlank
    @Size(min = 2, max = 32)
    private String content;
    private String country;
    private String method;
}

File: 151004/Korol/Lab1/DC_Lab1/DC_Lab1/DTO/MessageRequestTo.cs
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class MessageRequestTo(int Id,int? storyId, string? Content) : IRequestTo
    {
        public int Id { get; set; } = Id;
        public int? storyId { get; set; } = storyId;
        public string? Content { get; set; } = Content;
    }
}
